### PR TITLE
Feature/send errors

### PR DESCRIPTION
**Pull request by Cancuuu**
### Type of changes included:

- [x] Components
- [ ] Business Logic
- [x] Bug fixes
- [x] Styling
- [ ] Code Refactor

### Clubhouse Tickets Related:

- Ticket number: #532 

### Screenshots/Gifs:
Aug-09-2022 20-06-41

### Notes:

*Place special notes here*

### Tested on:

- [x] Chrome
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

**Comment by Cancuuu** [comment]
Guys i need help with the testing of the other errors, i only tested InsufficientBalance error

File: source/Popup/Views/SendError/index.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import { useSelector } from 'react-redux';
+import useStyles from './styles';
+
+const emojiError = {
+  100: FlyBill,
+  300: NoMouthFace,
+  301: RedFlag,
+};
+
+export default function SendErrorScreen() {
```
Comment (tomiir) [inline source/Popup/Views/SendError/index.jsx:20]: I'd separate this into 2 to keep consistency with the rest of the code. 

```
const SendErrorScreen = () => { ... };
export default SendErrorScreen;
```

File: source/redux/send.js [hunk 2]
```diff
@@ -35,9 +35,34 @@ const DEFAULT_STATE = {
...
+];
+
+const matchErrors = (err) => {
+  for (const errors of errorsInfo) {
+    const isCurrentError = errors.errorNames.some((errorName) => err.includes(errorName));
+    if (isCurrentError) {
+      return errors.errorCode;
+    }
+  }
+  return 600;
```
Comment (tomiir) [inline source/redux/send.js:60]: You could do:
```
const matchErrors = (err) => {
  const error = errorsInfo.find((info) => info?.errorNames?.includes(err));
  return error?.errorCode || 600;
```
};

File: source/Popup/Views/SendError/index.jsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import { Layout } from '@components';
+import { useRouter } from '@components/Router';
+import NoMouthFace from '@assets/icons/nomouth-face.svg';
+import RedFlag from '@assets/icons/red-flag.svg';
+import FlyBill from '@assets/icons/fly-bill.svg';
+
+import { useSelector } from 'react-redux';
+import useStyles from './styles';
+
+const emojiError = {
```
Comment (tomiir) [inline source/Popup/Views/SendError/index.jsx:14]: constants should be uppercase `ERROR_EMOJI`

**Review by tomiir — COMMENTED** [review]
Some code style improvements but lgtm overall 🚀

**Review by angarita-dev — APPROVED** [review]
lgtm

**Pull request merged**